src/audio/asrc/asrc_farrow.c

```diff
@@ -273,14 +271,17 @@ enum asrc_error_code asrc_get_required_size(struct comp_dev *dev,
 
 	/* accumulate the size */
 	size = sizeof(struct asrc_farrow);
-	size += sizeof(int32_t *) * num_channels; /* pointers the the buffers */
-	/* size of the ring buffers */
-	size += buffer_length * num_channels * (bit_depth / 8);
+
 	/* size of the impulse response */
-	size += filter_length * sizeof(int32_t);
+	size += ASRC_MAX_FILTER_LENGTH * sizeof(int32_t);
 
-	*required_size = size;
+	/* pointers the the buffers */
```

Comment:
typo: "to the buffers"

Reply:
oops!

Reply:
I consider this as only remaining PR fix action, do you agree?

Reply:
Fixed now!